src/types/models/mod_version.rs

```diff
@@ -436,6 +436,7 @@ impl ModVersion {
         gd: Option<GDVersionEnum>,
         platforms: Vec<VerPlatform>,
         major: Option<u32>,
+        statuses: Vec<ModVersionStatusEnum>,
```

Comment:
hmm
judging from the uses of the api, i wonder if this could be cleaned up somewhat

like maybe
rename `ModVersion::get_latest_for_mod` -> `ModVersion::get_latest_for_mod_statuses`
and then two methods
- `ModVersion::get_latest_for_mod` gets the latest for all mod statuses (so it's just calling `mod_statuses` with the vec of all mod statuses)
- `ModVersion::get_latest_for_mod_status` takes an enum and constructs the vector (for use in accepted mods)

the last one is probably weird, but i'm not the biggest fan of the giant vector of mod statuses in the mod feedback endpoint

Reply:
ok